templates/settings/payment.php

```diff
@@ -7,47 +7,124 @@
  * @package dokan
  */
 $has_methods = false;
+$has_methods_dropdown = false;
 
 do_action( 'dokan_payment_settings_before_form', $current_user, $profile_info ); ?>
 
-<form method="post" id="payment-form"  action="" class="dokan-form-horizontal">
+<div class="dokan-payment-settings-summary">
+    <h2 id="vendor-dashboard-payment-settings-error"></h2>
+    <div class="payment-methods-listing-header">
+        <h2> <?php esc_html_e( 'Payment Methods', 'dokan-lite' ); ?></h2>
+        <div id="vendor-dashboard-payment-settings-toggle-dropdown">
+            <button id="toggle-vendor-payment-method-drop-down" class="dokan-btn dokan-btn-success"> <?php esc_html_e( 'Add Payment Method', 'dokan-lite' ); ?></button>
+            <div id="vendor-payment-method-drop-down-wrapper">
+                <div id="vendor-payment-method-drop-down">
+                    <?php
+                    $profile_methods = array_keys( $profile_info['payment'] );
+                    $unused_methods  = array_diff( $methods, $profile_methods );
 
-    <?php wp_nonce_field( 'dokan_payment_settings_nonce' ); ?>
+                    $mis_match_methods = [ 'stripe', 'moip' ];
+                    $mis_match_methods = array_filter( $mis_match_methods, function ( $key ) use( $profile_methods ) {
+                        return in_array( $key,  $profile_methods);
+                    } );
+
+                    $unused_methods = array_filter( $unused_methods, function ( $key ) use ( $mis_match_methods ) {
+                        $found = false;
+
+                        foreach ( $mis_match_methods as $mis_match_method ) {
+                            $found = $found || ( false !== stripos( $key, $mis_match_method ) );
+                        }
+
+                        return ! $found;
+                    } );
+                    ?>
+                    <ul>
+                    <?php foreach ( $unused_methods as $method_key ) {
+                        $method = dokan_withdraw_get_method( $method_key );
+
+                        if ( ! empty( $method ) ) {
+                            $has_methods_dropdown = true;
+                        } else {
+                            continue;
+                        }
```

Comment:
Read the 2nd comment at `#L67` and if that happens, this codes are redundant.
Rewrite this in the following way:
```
                        if ( empty( $method ) ) {
                            continue;
                        }
```
Also `$has_methods_dropdown` is not needed.